modules/core/include/opencv2/core/hal/intrin_cpp.hpp

```diff
@@ -574,6 +574,49 @@ OPENCV_HAL_IMPL_REDUCE_MINMAX_FUNC(v_reduce_min, std::min)
 For 32-bit integer and 32-bit floating point types. */
 OPENCV_HAL_IMPL_REDUCE_MINMAX_FUNC(v_reduce_max, std::max)
 
+/** @brief Count the 1 bits in the vector and return 4 values
+
+Scheme:
+@code
+{A1 A2 A3 ...} => popcount(A1)
+@endcode
+Any types but result will be in v_uint32x4*/
+template<typename _Tp, int n> inline v_uint32x4 v_popcount(const v_reg<_Tp, n>& a)
+{
+    static unsigned char popCountTable[] =
```

Comment:
Ah, right!  Thank you for the reply.
BTW, I really want to confirm about that point before moving further.

```intrin_cpp.hpp``` will be used when neither SSE2 or NEON is available.
The number I noted above is NOT the result from ```intrin_cpp.hpp```
Still, is it important to measure the performance of ```v_popcount``` from ```intrin_cpp.hpp``` ?
(I'm just confirming to make sure I'm on the same page.  Your point makes sense but sounds a bit awkward to me)